gtsfm/multi_view_optimizer.py

```diff
@@ -28,118 +29,130 @@ def __init__(
         all_intrinsics: List[Optional[gtsfm_types.CALIBRATION_TYPE]],
         absolute_pose_priors: List[Optional[PosePrior]],
         relative_pose_priors: Dict[Tuple[int, int], PosePrior],
         two_view_reports_dict: Optional[Dict[Tuple[int, int], TwoViewEstimationReport]],
         cameras_gt: List[Optional[gtsfm_types.CAMERA_TYPE]],
         gt_wTi_list: List[Optional[Pose3]],
-    ) -> Tuple[Delayed, Delayed, Delayed, list]:
+        images_graph: Optional[List[Delayed]] = None,
+    ) -> Tuple[Delayed, Delayed, Delayed, List[Delayed]]:
         """Creates a computation graph for multi-view optimization.
 
         Args:
             num_images: number of images in the scene.
-            keypoints_graph: keypoints for images, each wrapped up as Delayed.
-            i2Ri1_graph: relative rotations for image pairs, each value wrapped up as Delayed.
-            i2Ui1_graph: relative unit-translations for image pairs, each value wrapped up as Delayed.
-            v_corr_idxs_graph: indices of verified correspondences for image pairs, wrapped up as Delayed.
+            delayed_keypoints: keypoints for images, each wrapped up as Delayed.
+            delayed_i2Ri1s: relative rotations for image pairs, each value wrapped up as Delayed.
+            delayed_i2Ui1s: relative unit-translations for image pairs, each value wrapped up as Delayed.
+            delayed_v_corr_idxs: indices of verified correspondences for image pairs, wrapped up as Delayed.
             all_intrinsics: intrinsics for images.
-            absolute_pose_priors: priors on the camera poses (not delayed).
-            relative_pose_priors: priors on the pose between camera pairs (not delayed)
-            two_view_reports_dict: Dict of TwoViewEstimationReports after inlier support processor.
+            absolute_pose_priors: priors on the camera poses.
+            relative_pose_priors: priors on the pose between camera pairs.
+            two_view_reports_dict:
             cameras_gt: list of GT cameras (if they exist), ordered by camera index.
             gt_wTi_list: list of GT poses of the camera.
+            images_graph (optional): list of images. Defaults to None.
 
         Returns:
             The GtsfmData input to bundle adjustment, aligned to GT (if provided), wrapped up as Delayed.
             The final output GtsfmData, wrapped up as Delayed.
-            Dict of TwoViewEstimationReports after view graph estimation.
             List of GtsfmMetricGroups from different modules, wrapped up as Delayed.
         """
+        metrics = []
 
-        if self._run_view_graph_estimator and self.view_graph_estimator is not None:
+        if self.view_graph_estimator is not None:
             (
-                viewgraph_i2Ri1_graph,
-                viewgraph_i2Ui1_graph,
-                viewgraph_v_corr_idxs_graph,
-                viewgraph_two_view_reports_graph,
-                viewgraph_estimation_metrics,
+                viewgraph_i2Ri1s,
```

Comment:
technically these are delayed_ too, so what is the policy? when do we use delayed_ and when do we not?

Reply:
I think we have to decide on the naming convention of functions and variables and clean up the whole repo.

Reply:
So we create an issue?

Reply:
Created here: https://github.com/borglab/gtsfm/issues/522